apps/crn-server/test/data-providers/contentful/reminders.data-provider.test.ts

```diff
@@ -0,0 +1,1264 @@
+        expect(result).toEqual({
+          total: 1,
+          items: [sharePresentationReminder],
+        });
+      });
+
+      it('Should not fetch the reminder when it passed more than 72 hours of the end of the event', async () => {
+        const startDate = '2023-01-01T08:00:00Z';
+        const endDate = '2023-01-01T10:00:00Z';
+        jest.setSystemTime(DateTime.fromISO('2023-01-04T10:01:00Z').toJSDate());
+
+        const eventMockResponse = getContentfulReminderEventsCollectionItem();
+        eventMockResponse!.startDate = startDate;
+        eventMockResponse!.endDate = endDate;
+
+        const users = getContentfulReminderUsersContent();
+
+        contentfulGraphqlClientMock.request.mockResolvedValueOnce({
+          eventsCollection: {
+            items: [eventMockResponse],
+          },
+          users,
+        });
+
+        const result = await remindersDataProvider.fetch(fetchRemindersOptions);
+
+        expect(result).toEqual({
+          total: 0,
+          items: [],
+        });
+      });
+
+      it('Should not fetch the reminder when the event is a future event', async () => {
+        const startDate = '2023-01-01T08:00:00Z';
+        const endDate = '2023-01-01T10:00:00Z';
+        jest.setSystemTime(DateTime.fromISO('2022-12-31T10:00:00Z').toJSDate());
+
+        const eventMockResponse = getContentfulReminderEventsCollectionItem();
+        eventMockResponse!.startDate = startDate;
+        eventMockResponse!.endDate = endDate;
+
+        const users = getContentfulReminderUsersContent();
+
+        contentfulGraphqlClientMock.request.mockResolvedValueOnce({
+          eventsCollection: {
+            items: [eventMockResponse],
+          },
+          users,
+        });
+
+        const result = await remindersDataProvider.fetch(fetchRemindersOptions);
+
+        expect(result).toEqual({
+          total: 0,
+          items: [],
+        });
+      });
+
+      it('Should not fetch the reminder if it has not ended', async () => {
+        const startDate = '2023-01-01T08:00:00Z';
+        const endDate = '2023-01-01T10:00:00Z';
+        jest.setSystemTime(DateTime.fromISO('2023-01-01T09:00:00Z').toJSDate());
+
+        const eventMockResponse = getContentfulReminderEventsCollectionItem();
+        eventMockResponse!.startDate = startDate;
+        eventMockResponse!.endDate = endDate;
+
+        const users = getContentfulReminderUsersContent();
+
+        contentfulGraphqlClientMock.request.mockResolvedValueOnce({
+          eventsCollection: {
+            items: [eventMockResponse],
+          },
+          users,
+        });
+
+        const result = await remindersDataProvider.fetch(fetchRemindersOptions);
+
+        expect(result.items.map((r) => r.type)).not.toContain(
+          'Share Presentation',
+        );
+      });
+    });
+
+    describe('Publish Material Reminder', () => {
+      describe('When less than 72 hours have passed since the event ended', () => {
```

Comment:
i think this describe is unnecessary (or simply wrong) because later you test cases for more than 72 hours have passed

Reply:
removed the describes here [remove nested describes indicating time since reference date](https://github.com/yldio/asap-hub/pull/3236/commits/5bc3a9f2789a14eb5138bdfe61d1c836a628df13)